### PR TITLE
chore(avatar, avatar-image): move into GlobalHeader component

### PR DESCRIPTION
**Pull request by dierat**
### Summary:
Follow-up to https://github.com/chanzuckerberg/edu-design-system/pull/1368/ in which the `Avatar` and `AvatarImage` components were moved from `src/components/` to `.storybook/recipes/`. In this PR, I'm moving the avatar svg code into the `GlobalHeader`, the only place where the `Avatar` and `AvatarImage` components are used, and deleting the avatar components entirely.

### Test Plan:
In storybook:
- `Avatar` and `AvatarImage` are gone
- `GlobalHeader` is unchanged

### CODE REVIEW

File: .storybook/recipes/GlobalHeader/GlobalHeader.module.css [hunk 1]
```diff
@@ -130,3 +130,33 @@
...
+
+/* Body of the avatar image. */
+.global-header__avatar-image__body {
+  height: 13px;
+}
+
+/* Legs of the avatar image. */
+.global-header__avatar-image__legs {
+  height: 10px;
+}
```
Comment (dierat) [inline .storybook/recipes/GlobalHeader/GlobalHeader.module.css:162]: These styles were copied wholesale with no changes.

File: .storybook/recipes/GlobalHeader/GlobalHeader.tsx [hunk 2]
```diff
@@ -51,8 +50,44 @@ export interface Props {
   curriculumIsActive?: boolean;
 }
 
+const Avatar = () => (
+  <div className={styles['global-header__avatar-image']}>
+    <svg
```
Comment (dierat) [inline .storybook/recipes/GlobalHeader/GlobalHeader.tsx:55]: From here down the avatar code was copied wholesale with no changes.

File: .storybook/recipes/GlobalHeader/GlobalHeader.tsx [hunk 3]
```diff
@@ -51,8 +50,44 @@ export interface Props {
   curriculumIsActive?: boolean;
 }
 
+const Avatar = () => (
+  <div className={styles['global-header__avatar-image']}>
```
Comment (dierat) [inline .storybook/recipes/GlobalHeader/GlobalHeader.tsx:54]: In the avatar components, it was possible to pass in a custom `className`, but we weren't using it.

File: .storybook/recipes/GlobalHeader/GlobalHeader.tsx [hunk 4]
```diff
@@ -51,8 +50,44 @@ export interface Props {
...
+      <g>
+        <path d="M5.73,55.51,2.58,56H1.14l-.85.44L0,57.73s.72,1.72.91,2a15,15,0,0,0,2.26,1.51L6.72,62l5.58-.36L13.39.5,8.48,0,7.89,55.19Z"></path>
+        <path d="M34.6,55.39a3.3,3.3,0,0,1-.24,1.77,13.77,13.77,0,0,1-1.46,1.35l-1.55.65-1.81.48H22.17L22.56.82,26,.74l.72,52.74a5.79,5.79,0,0,0,1,.59c.13,0,2.34,0,2.34,0l2.17.55Z"></path>
+      </g>
+    </svg>
+  </div>
+);
+
 /**
- * Primary UI component for user interaction
```
Comment (dierat) [inline .storybook/recipes/GlobalHeader/GlobalHeader.tsx:55]: I'm trying to get rid of these useless component comments.

**Review by jinlee93 — APPROVED** [review]
Nice, thanks for the cleanup.

**Review by booc0mtaco — APPROVED** [review]

**Comment by dierat** [comment]
Thanks for reviewing!

**Pull request merged**